.github/workflows/release-branch.yml

```diff
@@ -0,0 +1,54 @@
+name: Release branch
```

Comment:
We can collapse these into just one workflow. Before the push-artifact-ids we had to do different logic per branch/pr/release in the action, but that action now handles all the accounting, so we don' have to duplicate it.

Reply:
Ah yeah, that's a good point that we just have one canonical version of this and don't need any of the fancy rollout stuff. Kind of autopiloted this heh.